src/core/markovProcessGraph.js

```diff
@@ -260,7 +259,16 @@ const CONTRIBUTION_RADIATION = EdgeAddress.fromParts([
 ]);
 const SEED_MINT = EdgeAddress.fromParts(["sourcecred", "core", "SEED_MINT"]);
 
-export type FibrationOptions = {|
+export type Arguments = {|
+  +weightedGraph: WeightedGraphT,
+  +participants: $ReadOnlyArray<Participant>,
+  +intervals: $ReadOnlyArray<Interval>,
+  +parameters: Parameters,
+|};
+
+export type Parameters = {|
```

Comment:
Heh. It _is_ a little weird to have one parameter called `Arguments` and
another parameter called `Parameters` (especially since most people seem
to use “argument” and “parameter” interchangeably, anyway). I’m fine
with this for now, and might come in with a better name if I can think
of one.

(Are these hyperparameters?)